_includes/manuals/nightly/1.2_release_notes.md

```diff
@@ -6,6 +6,13 @@ This section will be updated prior to the next release.
 
 ### Upgrade warnings
 
+#### PostgreSQL upgraded to 13 on Enterprise Linux 8
+
+On Enterprise Linux 8 PostgreSQL is being upgraded from 12 to 13, to match the version used on Enterprise Linux 9.
+The actual upgrade, including data migration, happens when the Installer is executed for the first time.
```

Comment:
Would it make sense to tell users that there is an additional step in the upgrade instructions that doesn't perform the data migration?

```suggestion
Users are required to change the DNF module, but the actual upgrade, including data migration, happens when the installer is executed for the first time.
```